sdk/identity/identity/tsconfig.json

```diff
@@ -1,23 +1,9 @@
 {
   "extends": "../../../tsconfig",
-  "compilerOptions": {
-    "lib": ["DOM"],
-    "resolveJsonModule": true,
```

Comment:
I'm curious, why is `"resolveJsonModule": true` not added in the other `tsconfig` file?

Reply:
It is not needed.